src/controllers/userController.js

```diff
@@ -37,7 +36,7 @@ class userController {
       const Token = encode({
```

Comment:
can encode the whole user object(excluding the password) in the token and make sure that it's returned too (return the token)

Reply:
sure